core/src/main/kotlin/ch/ergon/dope/resolvable/clause/ISelectClause.kt

```diff
@@ -30,34 +30,37 @@ import ch.ergon.dope.validtype.NumberType
 import ch.ergon.dope.validtype.StringType
 import ch.ergon.dope.validtype.ValidType
 
-interface ISelectOffsetClause : Clause
+interface ISelectOffsetClause<T : ValidType> : Clause {
```

Comment:
Please update the diagram then as well (removing the inheritance from TypeExpression)

Reply:
done